.github/workflows/frontend.yml

```diff
@@ -6,32 +6,30 @@ on:
   push:
     branches: [main]
 
-defaults:
-  run:
-    working-directory: frontend
```

Comment:
This would effectively do a `cd frontend/` at the start of the Action - however this doesn't work anymore using npm workspaces as `npm ci` only works from the root where `package-lock.json` is located.

Reply:
Gotcha